exporter/splunkhecexporter/example/splunk.yml

```diff
@@ -4,8 +4,8 @@ splunk:
       directory: /opt/splunk/etc/apps/search/local
       content:
         metrics:
-          coldPath: $SPLUNK_DB/metrics/colddb
+          coldPath: ${env:SPLUNK_DB}/metrics/colddb
```

Comment:
This is a yml file used to configure Splunk Entreprise and is not related to Otel. Please revert those changes.

Reply:
Thanks, good catch! Fixed on the latest commit